Dockerfile

```diff
@@ -60,7 +60,7 @@ COPY --chown=axom:axom . /home/axom/workspace
 WORKDIR /home/axom/workspace
 RUN mkdir build && cd build && cmake -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_BUILD_TYPE=Debug -DENABLE_OPENMP=On -DCMAKE_CXX_FLAGS=-fsanitize=address ..
 RUN cd build && make -j 16
-RUN cd build && ./bin/raja-perf.exe --checkrun -sp
+RUN cd build && ./bin/raja-perf.exe --checkrun
```

Comment:
Why remove the '-sp' option? It's useful to capture the execution progress in the logs. In case of failure, it shows where to look.

Reply:
I'm just trying to trigger the azure CI

Reply:
Gotcha.